scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -6,5 +6,5 @@ module.exports = {
   // need for frequent changes as bundle sizes fluctuate.
 
   MIN_SIZE: 763,
-  MAX_SIZE: 1202,
+  MAX_SIZE: 1221,
```

Comment:
I'd try setting these to `768` and `1209`. Ran a `yarn build` locally on this branch and those should be calculated min/max values after the `lru-cache` update.

Reply:
ok, I followed the guidance on adding 5, as my figure for burmese was 1216. Were you getting lower figure?

Reply:
Yea I got `1204` in my local build 🤷🏼